platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdminExt.java

```diff
@@ -604,7 +620,12 @@ public Map getFactoryPidObjectClasses() {
             }
         }
 
-        return serviceList;
+        List<Map<String, Object>> authorizedServices = new ArrayList<>();
```

Comment:
Here, you're about 2/3 of the way to what I suggested above. Get rid of the `authorizedServices` and change the `forEach` to a collect.